CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+5. **Make changes**: Make your changes to the codebase.
+6. **Generate profiles**: Run `npm run generate` to generate the profiles.
+7. **Format style**: Run `npm run format` to format the style.
+8. **Run tests**: Run `npm test` to run the tests.
+9. **Commit changes**: Commit your changes to your forked repository.
+10. **Create a pull request**: Create a pull request from your forked repository to the appropriate branch of the main repository (see the Branching Model above).
+
+When a pull request is created, it will be reviewed by the Data Package maintainers. Github Automation creates a live preview site for every new pull request. Once the pull request is approved, it will be merged into the main repository.
+
+Note that the project uses two different types of produced artifacts:
+
+- **npm run generate**: This command generates the profiles from the `profiles` folder and sync profile version in the specs. It is required to run this command after changing the YAML/JSON files in `profiles` folder, and the output has to be committed to the repository.
+
+- **npm run build**: This command builds the project and generates the static site. This command is run by automation and the output is not commited to the repository. As a contributor, you don't need to run this command although you can use `npm run preview` to debug the site built in production-mode locally.
+
+## Backward compatibility
+
+This section outlines the rules for backward compatibility in the Data Package specification.
+
+1. An existing `datapackage.json` that is valid MUST NOT becoming invalid in the future.
+2. A new `datapackage.json` MAY be invalid because a software implementation does not support the latest version of the specification (yet).
+
+### Versioning
+
+1. The Data Package specification is versioned. This is new over 1.0, where changes were added without increasing the version.
+2. The Data Package specification is versioned as a whole: a number of changes are considered, discussed, added or refused and released as a new minor version.
+
+### Property changes
+
+1. A property MUST NOT change `type`
+2. A property MAY allow additional `type` (array)
+3. A property MUST NOT become `required`
+4. A property MAY become optional. Example: https://github.com/frictionlessdata/datapackage/pull/7
+5. A property MUST NOT add `enum`
+6. A property MAY remove `enum`. Example: https://github.com/frictionlessdata/specs/pull/809
+7. A property MUST NOT remove `enum` values
+8. A property MAY add `enum` values
+
+### Table schema changes
+
+1. A field type MUST NOT change default `format`. Example: does https://github.com/frictionlessdata/datapackage/pull/23 align with this?
+2. A field type MUST NOT remove `format` pattern options
+3. A field type MAY add `format` pattern options
+
+### New properties
+
+1. A new property MAY make a `datapackage.json` invalid (because of general rule 2). Example: https://github.com/frictionlessdata/datapackage/pull/24
```

Comment:
Is this not a contradiction of the backwards comparability rule?

> An existing datapackage.json that is valid MUST NOT becoming invalid in the future.

Reply:
Aah, I see now. It's only invalid because the validation software is not up to date with the latest standard. 

So while the `datapackage.json` is actually valid, it will not pass validation because the validator needs updating. Got it!